qiita_pet/static/js/sampleTemplateVue.js

```diff
@@ -435,6 +464,26 @@ Vue.component('sample-template-page', {
         // Add the select to update the sample information
         $row = $('<div>').attr('id', 'update-st-div').addClass('row form-group').appendTo($tab);
         $('<label>').addClass('col-sm-2 col-form-label').append('Update sample information:').appendTo($row);
+
+        // Add the direct upload field
+        $('<label>')
+          .addClass('btn btn-default')
+            .append('Direct upload file <small>(< 2MB)</small>')
+            .appendTo($row)
+            .append('<input type="file" style="display: none;" id="st-direct-upload">');
+        $('#st-direct-upload').on('change', function() {
+          if (this.files.length != 1) {
+            alert('You can only upload one file.')
+            return false;
+          }
+          if (this.files[0].size > 2097152) {
+            alert('You can only upload files smaller than 2MB; please use "Upload Files" button on the left.');
```

Comment:
What about something like this:

```suggestion
            alert('You can only upload files smaller than 2MB. For larger files please use the "Upload Files" button on the left.');
```